src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts

```diff
@@ -1,172 +1,76 @@
-import Ajv from "ajv";
-import schema from "./schema.json";
-import sampleDocument from "./certificate-of-origin-wrapped.json";
-import { cloneDeep, omit } from "lodash";
+import CertificateOfOriginSchema from "./CertificateOfOrigin.v3.json";
+import CertificateOfOriginSampleData from "./certificate-of-origin-data.json";
+import InvoiceSchema from "./Invoice.v3.json";
+import InvoiceSampleData from "./invoice-data.json";
+import { expand, JsonLdDocument } from "jsonld";
 
-const ajv = new Ajv({ allErrors: true });
-const validator = ajv.compile(schema);
-
-describe("certificate of origin schema", () => {
-  it("should work with valid certificate of origin", () => {
-    expect(validator(sampleDocument)).toBe(true);
+const expandDocument = async (mergedDocument: JsonLdDocument) => {
+  return await expand(mergedDocument, {
+    expansionMap: function(info) {
+      if (info.unmappedProperty) {
+        throw new Error(
+          '"The property ' +
+            (info.activeProperty ? info.activeProperty + "." : "") +
+            info.unmappedProperty +
+            ' in the input was not defined in the context"'
+        );
+      }
+    }
   });
+};
 
-  it("should fail when document does not have supplyChainConsignment", () => {
-    const document = omit(cloneDeep(sampleDocument), "supplyChainConsignment");
-
-    expect(validator(document)).toBe(false);
-    expect(validator.errors).toMatchInlineSnapshot(`
-      Array [
-        Object {
-          "dataPath": "",
-          "keyword": "required",
-          "message": "should have required property 'supplyChainConsignment'",
-          "params": Object {
-            "missingProperty": "supplyChainConsignment",
-          },
-          "schemaPath": "#/required",
-        },
-      ]
-    `);
+describe("certificate of origin schema", () => {
+  it("should work with valid certificate of origin", async () => {
```

Comment:
this does not test the schema but the context

Reply:
okay, i will change it to context.test.ts

Reply:
done. updated description from schema to context